Sensor Analysis/sensoranalysis.py

```diff
@@ -109,29 +93,14 @@ def main():
         "gas_data.timestamp": arr_df[:,71],
 
     }
-
-    items_to_find = ['False']
-    data_i = dict_sensor.get("has_lowG_data")
-    data_i2 = dict_sensor.get("has_highG_data")
-    data_i3 = dict_sensor.get("has_gps_data")
-    data_i4 = dict_sensor.get("has_barometer_data")
-    data_i5 = dict_sensor.get("has_kalman_data")
-    data_i6 = dict_sensor.get("has_rocketState_data")
-    data_i7 = dict_sensor.get("has_flap_data")
-    data_i8 = dict_sensor.get("has_voltage_data")
-    data_i9 = dict_sensor.get("has_orientation_data")
-    data_i10 = dict_sensor.get("has_magnetometer_data")
-    data_i11 = dict_sensor.get("has_gas_data")
+    # stuff at the bottom isnt't used anywhere in this file. dont want to delete until i double check. update: made a separate commit taking this out
```

Comment:
if you have any notes leave them in your commits or your PR comments rather than the actual code